docs/content-guidelines.md

```diff
@@ -0,0 +1,198 @@
+
+## Format
+
+This section provides you with guidelines on how to format and organize your text.
+
+### Headings
+
+Use H1 headings (preceded by # in Markdown) only at the start of the document to indicate the document name.
+
+Only use H2 and H3 headings to organize your content into categories. Do not use H4 (####) headings or lower.
+
+### Lists
+
+Lists are very useful for breaking up text and providing instructions.
+
+- Use bullet points (created with * or -) for lists involving general explanations.
+- Use numbered lists only for step-by-step instructions.
+
+### Tables
+
+Use tables when content needs comparison or to organize small bits of information. Keep in mind that in Markdown it is not possible to break up the content of a table cell into separate lines or bullet points, therefore tables are not useful when they contain long sentences or lists.
+
+You can find an example of a table in the [Styles](#styles) section of this document.
+
+### Code snippets
+
+Code snippets must be surrounded with the Markdown code block tag and the programming language should be specified. Make sure to indent code correctly using your text editor (2 space indentation is the default).
+
+For example:
+
+```javascript
+Luigi.setConfig({
+  routing: {
+    nodeParamPrefix: '~'
+  },
+```
+
+is better than:
+
+```
+Luigi.setConfig({
+    routing: {
+        nodeParamPrefix: '~'
+    },
+```
+
+### Styles
+
+Depending on the type of content, you should use different types of text, for example **bolded** or `code`.
+
+|     Type           |      Font          |    Example    |
+|--------------------|:------------------:|---------------|
+| Parameters         |      **bold**     | **viewGroup** |
+| Attributes | **bold** | **collapsible** |
+| Values         |      `code`     | `true`, `false` |
+| Dynamic parameters         |      `code`        | `{userId}`    |
+| Folders, paths, filenames  |      `code`        | Open `basicConfiguration.js` inside `assets/luigi-config`|
+| Code snippets      |      `code`        | See [this section](#code-snippets) |
+| Functions          |      `code`     | `showLoadingIndicator()` |
+
+## Links
+
+If the link is within the same folder on GitHub, use only the relative path. For all other links, including external links, use the absolute path (which starts with https://...).
+
+To link to a section within a document, use the title of that section as an anchor. For example:
+
+```
+[Link to the "custom messages" section in the "communication" document](communication.md#custom-messages)
+```
+
+## Screenshots and diagrams
+
+When adding screenshots or diagrams you should adhere to to the following rules:
+
+- Only use high resolution images
+- Only use screenshots or diagrams if necessary, as too many of them can create visual noise
+- As an alternative to screenshots, point to the [Luigi fiddle](https://fiddle.luigi-project.io/) or other Luigi examples when you want to illustrate a concept.
```

Comment:
```suggestion
- As an alternative to screenshots, point to the [Luigi fiddle](https://fiddle.luigi-project.io/) or other Luigi examples when you want to illustrate a concept.
```
is it `luigi fiddle` or `the luigi fiddle`?

Reply:
I changed it to "Luigi Fiddle"

Reply:
I changed it to "Luigi Fiddle"